### DISCUSSION TITLE
Restrict access to specific IP addresses

### DISCUSSION THREAD

**Discussion by abraralmadi** [globaleaks/globaleaks-whistleblowing-software#4546] [Q&A]
**Describe the bug**
when we Restrict access to specific IP addresses for the admin user even the user with the ip can't access the site 

**To Reproduce**
1. Go to '.admin/network'
2. Click on 'Access control'
3. Scroll down to 'check admin add your IP'SAVE 
4. Then try access See error

**Expected behavior**
log in but it does not 
**Screenshots**

**Desktop (please complete the following information):**
 - OS: [mac]
 - Version [3.11.69 ]

**Additional context**
is there a way to retrieve the setting to default from the console?or any other alternative solution

**Comment by abraralmadi** [comment]
2021-03-25T11:25:49+0000 [twisted.python.log#info] (ip) [REMOVED_IP_ADDRESS] - - [25/Mar/2021:11:25:48 +0000] "POST /authentication HTTP/1.1" 401 105 1328ms - "[REMOVED_USER_AGENT]" 1

**Comment by evilaliv3** [comment]
Thank you @abraralmadi 

I know that you are using docker and probably GlobaLeaks in your setup for one reason or an other is not receiing the original user ip.

To reset this configuration you could issue: ```gl-admin setvar ip_filter_admin ""```

**Comment by abraralmadi** [comment]
@evilaliv3 thank you so much you are the best person on the planet

**Comment by abraralmadi** [comment]
@evilaliv3 is there a command we can disable the certificate using it ? because the admin cant access the and the error with no explanation

**Comment by evilaliv3** [comment]
```gl-admin setvar https_enabled false ```

**Comment by abraralmadi** [comment]
i am sorry but last question is there a way i can know all the cases that the  the user will git error with no explanation? so i can troubleshoot?

**Comment by evilaliv3** [comment]
Thank you for your question but actually not such a request not exists.

Please open separated tickets for separated topics. It makes more easy for
the whole community to answer and find answers. Thank you

**Comment by abraralmadi** [comment]
thank you
